bot/cogs/tags.py

```diff
@@ -86,11 +86,58 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
             return self._get_suggestions(tag_name)
         return found
 
+    async def _get_tags_via_content(self, check: Callable[[Iterable], bool], keywords: str) -> Optional[Embed]:
+        """
+        Search for tags via contents.
+
+        `predicate` will be either any or all, or a custom callable to search. Must return a bool.
+        """
+        await self._get_tags()
+
+        keywords_processed: Tuple[str] = tuple(query.strip().casefold() for query in keywords.split(','))
+        founds: list = [
+            tag
+            for tag in self._cache.values()
+            if check(query in tag['embed']['description'] for query in keywords_processed)
+        ]
+
+        if not founds:
+            return None
+        elif len(founds) == 1:
+            return Embed().from_dict(founds[0]['embed'])
+        else:
+            return Embed(
+                title='Did you mean ...',
```

Comment:
This title is a bit misleading. It may be interpreted as a list of search _terms_ being suggested, rather than a list of search _results_ (tag names). It should be clear what is being listed are tag names results for the search.

Reply:
Or another way to look at it is that it may suggest that the command is meant to search the names of tags rather than their contents.

Reply:
Hmm, that is a good catch, it does sound like the following *terms* are what you are looking for.
How does `Are you perhaps looking for the following tags?` sound?
Or maybe a simple `Are you looking for these tags?`?

Reply:
`The following tags have {keywords} in their description, perhaps you are looking for one of these?` sounds pretty descriptive, but it might be too long, what do you think?

Reply:
That sounds OK. What do you think of  `Here are the tags containing the given keyword(s):`?

Reply:
Yep, that definitely sounds better and shorter, thank you!

Reply:
`Are you looking for these tags?` was fine too. I don't think the actual keywords should be included as it may make the title too long.